dashboards/google-kubernetes-engine/gke-optimization-signals.json

```diff
@@ -210,7 +209,7 @@
                 "plotType": "LINE",
                 "targetAxis": "Y1",
                 "timeSeriesQuery": {
-                  "timeSeriesQueryLanguage": "{ fetch k8s_container::kubernetes.io/container/cpu/request_cores;\n fetch k8s_node::kubernetes.io/node/cpu/allocatable_cores\n} \n| ${project_id}\n| ${location}\n| ${cluster_name}\n| join\n| group_by [project_id, location, cluster_name], sum(val(0)) / sum(val(1))",
+                  "timeSeriesQueryLanguage": "{ fetch k8s_container::kubernetes.io/container/cpu/request_cores\n| filter\n    resource.namespace_name !~ '(kube|istio|gatekeeper|gke|gmp|gke-gmp)-system|gke-managed-filestorecsi|gke-mcs'\n| ${project_id}\n| ${location}\n| ${cluster_name}\n    | group_by [], .aggregate()\n    ;\n fetch k8s_node::kubernetes.io/node/cpu/allocatable_cores\n| ${project_id}\n| ${location}\n| ${cluster_name}\n | group_by [], .aggregate()\n} \n| join\n| value val(0) / val(1)\n| scale '%'",
```

Comment:
If you make both the numerator and denominator `| group_by [resource.cluster_name, resource.location, resource.project_id]` as follows

![Screenshot 2024-03-18 at 10 21 15 AM](https://github.com/GoogleCloudPlatform/monitoring-dashboard-samples/assets/6540967/95cf4448-5f05-4ea4-969e-88be8cb24064)

You get the best of both worlds:
- when there are multiple clusters, you get one time series per cluster
- each time series is automatically labelled by the cluster name
- when the filters are drilled down to a single cluster, the single time series value matches the scorecard

And you should do this for Cluster Bin Packing Memory as well